packages/subgraph/tests/constants.ts

```diff
@@ -11,6 +11,7 @@ export const ADDRESS_FIVE = '0x0000000000000000000000000000000000000005';
 export const ADDRESS_SIX = '0x0000000000000000000000000000000000000006';
 export const DAO_ADDRESS = '0x00000000000000000000000000000000000000da';
 export const CONTRACT_ADDRESS = '0x00000000000000000000000000000000000000Ad';
+export const WRAPPED_CONTRACT_ADDRESS = '0x0000000000000000000000000000000000000Ad1';
```

Comment:
u can not use `DAO_TOKEN_ADDRESS`got wrapped token as well?

Reply:
I could use any of them, but I added it for the "readability" of the tests. If you prefer it, I can delete It and use any other address

Reply:
u can create `const WRAPPED_CONTRACT_ADDRESS = CONTRACT_ADDRESS` inside your individual test on in the global constants as u wish